### PR TITLE
[DEV APPROVED] TP 7710 mobile footer optimisation

### PR DESCRIPTION
**Pull request by davidtrussler**
The purpose of this is to optimise the footer on smaller devices to avoid the need for the user to scroll too far. There are some additional changes in PR#20 on Yeast (https://github.com/moneyadviceservice/yeast/pull/20)

Current footer:
image

Footer updated by this PR: 
image

---
This change is [Reviewable](https://reviewable.io/reviews/moneyadviceservice/frontend/1583)

### CODE REVIEW

**Comment by slaywell** [comment]
Looking good! Couple of initial comments before I look at the code: 

1. Please could you adjust the 'Web Chat' / 'Call Us' text so that the icon sits vertically in the centre of the line, not at the baseline?
2. The centre alignment on the About Us / Media Centre etc. looks odd. With this section I would take the same approach you have done with the Sign in etc. links right at the bottom - float them left with the vertical line to separate them.

**Comment by davidtrussler** [comment]
@slaywell I've updated those couple of things (and the screenshot).

File: app/assets/stylesheets/components/common/_icon.scss [hunk 1]
```diff
@@ -35,6 +35,8 @@
 // Styleguide icons
 
 //= depend_on_asset icon-sprite.png
+$icon-scale: 0.625;
+$icon-sprite-bg-size: $icon-scale * $sprite-background-size * 2;
```
Comment (slaywell) [inline app/assets/stylesheets/components/common/_icon.scss]: Are these unused now that you've refactored the sprites?

File: app/views/shared/_contact_panels.html.erb [hunk 2]
```diff
@@ -49,13 +51,15 @@
...
-        <p class="t-contact-introduction"><%= @footer.contact.introduction %></p>
+        <p class="t-contact-introduction contact-panel__contact-introduction"><%= @footer.contact.introduction %></p>
 
-        <p>
+        <p class="contact-panel__contact-number">
           <a class="contact-panel__telephone t-contact-number" href="<%= t('contact.telephone_number_href') %>">
             <%= @footer.contact.phone_number %></a>
         </p>
 
+        <p class="contact-panel__opening-times-heading">Open:</p>
```
Comment (slaywell) [inline app/views/shared/_contact_panels.html.erb]: Not sure that these should be `<p>` tags - is there a more semantic element?

File: app/assets/stylesheets/components/common/_icon.scss (hunk 1 above)
Comment (davidtrussler) [inline app/assets/stylesheets/components/common/_icon.scss]: Yep, true dat. Will delete.

File: app/views/shared/_footer_secondary.html.erb [hunk 3]
```diff
@@ -33,46 +32,50 @@
...
-
+      <ul class="footer-secondary__list footer-secondary__link-items">
         <% unless hide_elements_irrelevant_for_third_parties? %>
-          <li class="footer-secondary__list-item l-footer-authentication-link">
-            <% if user_signed_in? %>
+          <% if user_signed_in? %>
+            <li class="footer-secondary__list-item footer-secondary__link-item">
               <%= button_to t('authentication.sign_out'), main_app.destroy_user_session_path, class: 'authentication__link footer-secondary__link', method: :delete %>
-              <span class="footer-site-links__pipe">|</span>
+              <%# <span class="footer-site-links__pipe">|</span> %>
```
Comment (slaywell) [inline app/views/shared/_footer_secondary.html.erb]: Can we delete these comments?

**Comment by slaywell** [comment]
Only pretty minor comments - it's looking great.

File: app/views/shared/_footer_secondary.html.erb (hunk 3 above)
Comment (davidtrussler) [inline app/views/shared/_footer_secondary.html.erb]: Yes, just did that actually. Will push in a bit.

**Comment by rnalexander** [comment]
:+1:

**Comment by slaywell** [comment]
👍 👍

**Pull request merged**